packages/chakra-ui/src/BaseInputTemplate/BaseInputTemplate.tsx

```diff
@@ -56,11 +53,11 @@ export default function BaseInputTemplate<
       isReadOnly={readonly}
       isInvalid={rawErrors && rawErrors.length > 0}
     >
-      {displayLabel ? (
+      {displayLabel && !!label && (
         <FormLabel htmlFor={id} id={`${id}-label`}>
-          {label || schema.title}
```

Comment:
And since we don't render empty labels, we don't need the `|| schema.title` everywhere in the `chakra-ui` theme